task_05.py

```diff
@@ -61,12 +61,12 @@ def is_empty(my_sequence):
     """
     count = get_member_count(my_sequence)
 
-    if count != False:
+    if count != True:
```

Comment:
The text of the pylint error here was:

```
PEP8 E712: comparison to True should be 'if cond is True:' or 'if cond:'
```

That's because the `!=` operator returns `True` or `False` so it's redundant if the variable you're testing is already `True` or `False` -- eg, `if count:` is enough